BitwardenShared/UI/Vault/VaultItem/ViewItem/ViewIdentityItem/ViewIdentityItemView.swift

```diff
@@ -12,82 +12,109 @@ struct ViewIdentityItemView: View {
 
     var body: some View {
         if !store.state.identityName.isEmpty {
+            let identityName = store.state.identityName
             BitwardenTextValueField(
                 title: Localizations.identityName,
-                value: store.state.identityName,
-                valueAccessibilityIdentifier: "ItemNameEntry"
+                value: identityName,
+                valueAccessibilityIdentifier: "ItemNameEntry",
+                copyButtonAction: { store.send(.copyPressed(value: identityName, field: .identityName)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopyNameButton"
             )
             .accessibilityElement(children: .contain)
         }
 
         if !store.state.userName.isEmpty {
+            let username = store.state.userName
             BitwardenTextValueField(
                 title: Localizations.username,
-                value: store.state.userName,
-                valueAccessibilityIdentifier: "IdentityUsernameEntry"
+                value: username,
+                valueAccessibilityIdentifier: "IdentityUsernameEntry",
+                copyButtonAction: { store.send(.copyPressed(value: username, field: .username)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopyUsernameButton"
             )
             .accessibilityElement(children: .contain)
         }
 
         if !store.state.company.isEmpty {
+            let company = store.state.company
             BitwardenTextValueField(
                 title: Localizations.company,
-                value: store.state.company,
-                valueAccessibilityIdentifier: "IdentityCompanyEntry"
+                value: company,
+                valueAccessibilityIdentifier: "IdentityCompanyEntry",
+                copyButtonAction: { store.send(.copyPressed(value: company, field: .company)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopyCompanyButton"
             )
             .accessibilityElement(children: .contain)
         }
 
         if !store.state.socialSecurityNumber.isEmpty {
+            let socialSecurityNumber = store.state.socialSecurityNumber
             BitwardenTextValueField(
                 title: Localizations.ssn,
-                value: store.state.socialSecurityNumber,
-                valueAccessibilityIdentifier: "IdentitySsnEntry"
+                value: socialSecurityNumber,
+                valueAccessibilityIdentifier: "IdentitySsnEntry",
+                copyButtonAction: { store.send(.copyPressed(value: socialSecurityNumber, field: .socialSecurityNumber)) },
+                copyButtonAccessibilityIdentifier: "IdentityCopySocialSecurityNumberButton"
```

Comment:
🎨 As the value accessibility identifier we're using `IdentitySsnEntry` with "Social security number" as "Ssn", so unless QA automation folks think differently we should maintain the same approach here as well => "IdentityCopySsnButton".